.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
   },
```

Comment:
this change was automatically done by VScode. I am not sure this is backward compatible with other developers versions. Lets bring it up on the next meeting. For now please leave it as true